packages/core/src/projections/augury-event-projection.class.ts

```diff
@@ -0,0 +1,18 @@
+import { AuguryEvent } from '../events';
+import { Projection } from './projection.interface';
+
+export abstract class AuguryEventProjection<Output> implements Projection<AuguryEvent, Output> {
```

Comment:
Could we simplify this name to `EventProjection`? Ideally we can avoid using the term `Augury` in class names imo. Unfortunately the term `Event` is already used, but it would be ideal to avoid `AuguryEvent`. I guess we could have gone with something like `EventInfo` or `RawEvent`, etc.